src/write.ts

```diff
@@ -117,7 +117,58 @@ function getCurrentRepo() {
 }
 
 function floatStr(n: number) {
-    return Number.isInteger(n) ? n.toFixed(1) : n.toString();
+    return Number.isInteger(n) ? n.toFixed(0) : n.toFixed(2);
+}
+
+function strVal(b: BenchmarkResult): string {
+    let s = `\`${b.value}\` ${b.unit}`;
+    if (b.range) {
+        s += ` (\`${b.range}\`)`;
+    }
+    return s;
+}
+
+function commentFooter(): string {
+    const repo = getCurrentRepo();
+    // eslint-disable-next-line @typescript-eslint/camelcase
+    const repoUrl = repo.html_url ?? '';
+    const actionUrl = repoUrl + '/actions?query=workflow%3A' + encodeURIComponent(github.context.workflow);
+    core.debug(`Action URL: ${actionUrl}`);
```

Comment:
Would you remove this debug print? We can check the `actionUrl` value in comment. So printing it in job is not so useful.